packages/grafana-toolkit/src/cli/tasks/plugin.sign.ts

```diff
@@ -23,8 +23,10 @@ const pluginSignRunner: TaskRunner<PluginSignOptions> = async ({ signatureType,
       manifest.rootUrls = rootUrls;
     }
 
+    const pkg = await fs.readFile(`${__dirname}/../../../package.json`, 'utf8');
```

Comment:
Would this relative path always work?

Reply:
@marefr I hope they do, similar to what we use here https://github.com/grafana/grafana/blob/main/packages/grafana-toolkit/src/cli/index.ts#L135

Reply:
:+1:

Reply:
Maybe extract that call into a utility function and reuse it in both places to make it easier to change this in the future.

Reply:
@mckn good point, moved it into a util https://github.com/grafana/grafana/pull/44326/commits/0df66b4fe820636ca44483d549e8a43460481246